packages/replay/src/launchdarkly.ts

```diff
@@ -1,7 +1,7 @@
-import dbg from "./debug";
+import weakMemoize from "@emotion/weak-memoize";
```

Comment:
it's a fairly small package. We could inline it. I mostly have resorted to using the `WeakMap`-based memoization for now since this module acts as a singleton. With the proposed change and relying on dependency injection, we can no longer assume that it's just a singleton. 

It could be nice to move LauncDarkly's initialization closer to the `ctx` creation and just pass it around everywhere with it. This would address the issue and remove the need for this package (or its equivalent). Since the PR already touches a lot of files I decided to stay way from changing this as part of this PR